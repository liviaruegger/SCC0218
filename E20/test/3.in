10
bcbccbe b
bcebebeaae bc
abbbcebedc aba
ebdbdccaea d
dbceab b
bddecedb d
ccddedab ced
edbcacce dda
bdeaadcc d
adeba d
