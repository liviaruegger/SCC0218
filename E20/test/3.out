3
1 3 6 

1
1 

Not Found

2
3 5 

2
2 6 

3
2 3 7 

Not Found

Not Found

2
2 6 

1
2 

